.changeset/quick-papayas-study.md

```
@@ -0,0 +1,5 @@
+---
+"@assistant-ui/react": patch
+---
+
+refactor: ThreadList
```

Markdownlint (0.35.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)